books/management/commands/resync.py

```diff
@@ -71,7 +71,7 @@ def handle(self, *args, **options):
             if book:
                 if success:
                     counter['success'] = counter['success'] + 1
-                    self.stdout.write(self.style.SUCCESS(
+                    self.stdout.write(self.style.HTTP_REDIRECT(
```

Comment:
I'm not sure yet about what's the best way to output info. Identifier showing up during addepub I removed since it was my addition. I thought at one point unique identification by identifier rather than sha was a possibility, though I no longer consider that a good idea. I think outputting parsed authors and tags, later perhaps through some debug option, is helpful during development. I was also considering adding filenames to a list of imported/not imported that would be shown as a summary at the end. stdout.write vs logging or another method was also one of my questions. A good candidate for refactoring to functions so there'd be less typing each time one of the various output styles is used.